src/components/CardEditor/CardEditForm/CardEditForm.jsx

```diff
@@ -157,6 +159,44 @@ export const basicCardValidation = (card) => {
   return errors;
 };
 
+/**
+ * removes properties needed for the editor that we don't want the user to be able to modify
+ * @param {Object} card JSON currently being edited
+ * @returns {Object} card with omitted attributes
+ */
+export const hideCardPropertiesForEditor = (card) => {
+  let attributes;
+  let series;
+  if (card.content?.attributes) {
+    attributes = card.content?.attributes?.map((attribute) =>
+      omit(attribute, [
+        'aggregationMethods',
+        'aggregationMethod',
+        'grain',
+        'uuid',
+      ])
+    );
+  }
+  if (card.content?.series) {
+    series = card.content?.series?.map((attribute) =>
```

Comment:
the null checks for `card.contentseries` isn't need here since its already checked